library/src/main/java/com/datatorrent/lib/io/fs/FileSplitterInput.java

```diff
@@ -266,6 +266,7 @@ public TimeBasedDirectoryScanner getScanner()
   public static class TimeBasedDirectoryScanner implements Runnable, Component<Context.OperatorContext>
   {
     private static long DEF_SCAN_INTERVAL_MILLIS = 5000;
+    private static String FILE_BEING_COPIED = "_COPYING_";
```

Comment:
1. Ignore regex is a property that is a property is set by the user so it can be changed by the user anytime.
IMO this is a clean/explicit way of doing it.  

2. That is correct. However it is highly unlikely that on other file systems, the  actual files are called something that ends with _COPYING_. If its important I will make this run for only HDFS